Corrected reference

blubberbaer01 opened the pull request:
Since pycrypto is out-of-date

blubberbaer01 committed:
Corrected reference

Since pycrypto is out-of-date

pinae commented:
Could you please give some additional information why `pycrypto` is deprecated? The website does not mention it: https://www.dlitz.net/software/pycrypto/

blubberbaer01 commented:
Hi Pina,

I tried to follow the instructions in the README.md and got an error while pycrypto is getting compiled. I looked in GitHub for a solution and found the following comment https://github.com/pycrypto/pycrypto/issues/238 <https://github.com/pycrypto/pycrypto/issues/238>. I followed the instructions and replaced pycrypto with pycryptodome also in the requirement.txt and everything worked out fine then. I updated the file locally and sent you the request to accept the change in the requirement.txt.

Thanks
Thomas

P.S. In case of need I can send you the output of the command: pip install -r requirements.txt with pycrypto and pycryptodome. 

> Am 07.06.2020 um 11:30 schrieb Pina Merkert <notifications@github.com>:
> 
> 
> Could you please give some additional information why pycrypto is deprecated? The website does not mention it: https://www.dlitz.net/software/pycrypto/ <https://www.dlitz.net/software/pycrypto/>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/pinae/ctSESAM-python-memorizing/pull/7#issuecomment-640184657>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AA5S3DM7LZZYQRNKXDROCFLRVNM3FANCNFSM4NWFGI6A>.
>
